agents/postgres/pgstatsstatements/pgstatsstatements_test.go

```diff
@@ -0,0 +1,328 @@
+//				DigestText:      pointer.ToString("SELECT 'Same'"),
+//				CountStar:       10,
+//				SumRowsAffected: 50,
+//			},
+//		}
+//		actual := makeBuckets(current, prev, logrus.WithField("test", t.Name()))
+//		require.Len(t, actual, 0)
+//	})
+//
+//	t.Run("Truncate", func(t *testing.T) {
+//		prev := map[string]*pgStatStatements{
+//			"Truncate": {
+//				Digest:          pointer.ToString("Truncate"),
+//				DigestText:      pointer.ToString("SELECT 'Truncate'"),
+//				CountStar:       10,
+//				SumRowsAffected: 50,
+//			},
+//		}
+//		current := map[string]*pgStatStatements{}
+//		actual := makeBuckets(current, prev, logrus.WithField("test", t.Name()))
+//		require.Len(t, actual, 0)
+//	})
+//
+//	t.Run("TruncateAndNew", func(t *testing.T) {
+//		prev := map[string]*pgStatStatements{
+//			"TruncateAndNew": {
+//				Digest:          pointer.ToString("TruncateAndNew"),
+//				DigestText:      pointer.ToString("SELECT 'TruncateAndNew'"),
+//				CountStar:       10,
+//				SumRowsAffected: 50,
+//			},
+//		}
+//		current := map[string]*pgStatStatements{
+//			"TruncateAndNew": {
+//				Digest:          pointer.ToString("TruncateAndNew"),
+//				DigestText:      pointer.ToString("SELECT 'TruncateAndNew'"),
+//				CountStar:       5,
+//				SumRowsAffected: 25,
+//			},
+//		}
+//		actual := makeBuckets(current, prev, logrus.WithField("test", t.Name()))
+//		require.Len(t, actual, 1)
+//		expected := &qanpb.MetricsBucket{
+//			Queryid:          "TruncateAndNew",
+//			Fingerprint:      "SELECT 'TruncateAndNew'",
+//			AgentType:        inventorypb.AgentType_QAN_MYSQL_PERFSCHEMA_AGENT,
+//			NumQueries:       5,
+//			MRowsAffectedCnt: 5,
+//			MRowsAffectedSum: 25,
+//		}
+//		assertBucketsEqual(t, expected, actual[0])
+//	})
+//}
+
+func logTable(t *testing.T, structs []reform.Struct) {
+	t.Helper()
+
+	if len(structs) == 0 {
+		t.Log("logTable: empty")
+		return
+	}
+
+	columns := structs[0].View().Columns()
+	var buf bytes.Buffer
+	w := tabwriter.NewWriter(&buf, 0, 0, 1, ' ', tabwriter.Debug)
+	_, err := fmt.Fprintln(w, strings.Join(columns, "\t"))
+	require.NoError(t, err)
+	for i, c := range columns {
+		columns[i] = strings.Repeat("-", len(c))
+	}
+	_, err = fmt.Fprintln(w, strings.Join(columns, "\t"))
+	require.NoError(t, err)
+
+	for _, str := range structs {
+		res := make([]string, len(str.Values()))
+		for i, v := range str.Values() {
+			res[i] = spew.Sprint(v)
+		}
+		fmt.Fprintf(w, "%s\n", strings.Join(res, "\t"))
+	}
+
+	require.NoError(t, w.Flush())
+	t.Logf("%s:\n%s", structs[0].View().Name(), buf.Bytes())
+}
+
+func setup(t *testing.T, db *reform.DB) *PGStatStatementsQAN {
+	t.Helper()
+
+	_, err := db.Exec(`do $$
+	begin
+	perform pg_stat_statements_reset();
+	end
+	$$;`)
+	require.NoError(t, err)
+
+	return newPgStatsStatementsQAN(db, "agent_id", logrus.WithField("test", t.Name()))
+}
+
+// filter removes buckets for queries that are not expected by tests.
+func filter(mb []*qanpb.MetricsBucket) []*qanpb.MetricsBucket {
```

Comment:
All those things are MySQL-specific. Let's keep only `EXPLAIN` as an example, and remove everything else.